ingesters/winevents/main.go

```diff
@@ -169,46 +174,27 @@ func debugout(format string, args ...interface{}) {
 	fmt.Printf(format, args...)
 }
 
-type errWriter func(format string, args ...interface{})
-
-func interactiveErrorWriter(format string, args ...interface{}) {
-	fmt.Fprintf(os.Stderr, format, args...)
-}
-
-func serviceErrorWriter(format string, args ...interface{}) {
-	elog.Error(1, fmt.Sprintf(strings.Trim(format, "\n\r"), args...))
-}
-
-func interactiveWarnWriter(format string, args ...interface{}) {
-	fmt.Fprintf(os.Stderr, format, args...)
-}
-
-func serviceWarnWriter(format string, args ...interface{}) {
-	elog.Warning(1, fmt.Sprintf(strings.Trim(format, "\n\r"), args...))
-}
-
-func interactiveInfoWriter(format string, args ...interface{}) {
-	fmt.Printf(format, args...)
-}
-
-func serviceInfoWriter(format string, args ...interface{}) {
-	elog.Info(1, fmt.Sprintf(strings.Trim(format, "\n\r"), args...))
-}
-
-func errorout(format string, args ...interface{}) {
-	if errW != nil {
-		errW(format, args...)
-	}
-}
-
-func infoout(format string, args ...interface{}) {
-	if infW != nil {
-		infW(format, args...)
-	}
+type levelLogger struct {
+	elog debug.Log
 }
 
-func warnout(format string, args ...interface{}) {
-	if warnW != nil {
-		warnW(format, args...)
+// levelLogger implements the log.LevelRelay interface
+func (l levelLogger) WriteLog(lvl log.Level, ts time.Time, msg []byte) error {
+	switch lvl {
+	case log.DEBUG:
+		if verbose {
+			fmt.Fprintln(os.Stdout, string(msg))
```

Comment:
Stdout or Stderr?

Reply:
i think for pure debug it should be stdout.  I think the O.G. choice of stderr was incorrect